tests/admin_docs/test_views.py

```diff
@@ -339,4 +339,5 @@ def test_simplify_regex(self):
             (r'^a/?$', '/a/'),
         )
         for pattern, output in tests:
-            self.assertEqual(simplify_regex(pattern), output)
+            with self.subTest(pattern=pattern, output=output):
```

Comment:
In cases like this, I don't think there's a need to include `output` as a `subTest()` argument since `pattern` identifies each test uniquely and `output` will appear in the assertion error if the test fails. Could you review the PR with this idea in mind?